src/test/java/htsjdk/samtools/cram/structure/DataSeriesTest.java

```diff
@@ -0,0 +1,39 @@
+package htsjdk.samtools.cram.structure;
+
+import htsjdk.HtsjdkTest;
+import htsjdk.samtools.cram.CRAMException;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class DataSeriesTest extends HtsjdkTest {
+
+    @DataProvider(name = "twoCharsPositive")
+    public static Object[][] twoCharsPositive() {
+        return new Object[][] {
+                {"RN", DataSeries.RN_ReadName},
+                {"QS", DataSeries.QS_QualityScore},
+                {"BBBBBBBB", DataSeries.BB_bases}
```

Comment:
It's weird to accept this.  Maybe we should assert that it actually is 2chars long?  Does it matter at all?

Reply:
True.  It's strictly 2 chars in the spec, and that is now enforced.